test/dataframe_builders_test.ml

```diff
@@ -0,0 +1,127 @@
+open! Core
+open Polars
+
+let%expect_test "Basic clear" =
+  let df =
+    Data_frame.create_exn
+      Series.[ int "integer" [ 1; 2; 3; 4; 5 ]; float "float" [ 4.; 5.; 6.; 7.; 8. ] ]
+  in
+  let df2 = Data_frame.clear df in
+  Data_frame.to_string_hum df |> print_endline;
+  Data_frame.to_string_hum df2 |> print_endline;
+  [%expect
+    {|
+    shape: (5, 2)
+    ┌─────────┬───────┐
+    │ integer ┆ float │
+    │ ---     ┆ ---   │
+    │ i64     ┆ f64   │
+    ╞═════════╪═══════╡
+    │ 1       ┆ 4.0   │
+    │ 2       ┆ 5.0   │
+    │ 3       ┆ 6.0   │
+    │ 4       ┆ 7.0   │
+    │ 5       ┆ 8.0   │
+    └─────────┴───────┘
+    shape: (0, 2)
+    ┌─────────┬───────┐
+    │ integer ┆ float │
+    │ ---     ┆ ---   │
+    │ i64     ┆ f64   │
+    ╞═════════╪═══════╡
+    └─────────┴───────┘ |}]
+;;
+
+let%expect_test "Basic vstack" =
+  let df =
+    Data_frame.create_exn
+      Series.[ int "integer" [ 1; 2; 3; 4; 5 ]; float "float" [ 4.; 5.; 6.; 7.; 8. ] ]
+  in
+  let df2 =
+    Data_frame.create_exn
+      Series.[ int "integer" [ 1; 2; 3; 4; 5 ]; float "float" [ 4.; 5.; 6.; 7.; 8. ] ]
+  in
+  Data_frame.vstack_exn df ~other:df2;
+  Data_frame.to_string_hum df |> print_endline;
+  [%expect
+    {|
+    shape: (10, 2)
+    ┌─────────┬───────┐
+    │ integer ┆ float │
+    │ ---     ┆ ---   │
+    │ i64     ┆ f64   │
+    ╞═════════╪═══════╡
+    │ 1       ┆ 4.0   │
+    │ 2       ┆ 5.0   │
+    │ 3       ┆ 6.0   │
+    │ 4       ┆ 7.0   │
+    │ …       ┆ …     │
+    │ 2       ┆ 5.0   │
+    │ 3       ┆ 6.0   │
+    │ 4       ┆ 7.0   │
+    │ 5       ┆ 8.0   │
+    └─────────┴───────┘ |}]
+;;
+
+let build_a_dataframe_usage n =
```

Comment:
Let's leave a simple explanation of the what these benchmarks are supposed to simulate and motivation (quadratic runtime of certain operations)

Reply:
done